src/cmd_line.rs

```diff
@@ -32,6 +33,7 @@ impl Default for CmdLineSettings {
             remote_tcp: None,
             multi_grid: false,
             maximized: false,
+            frameless: true,
```

Comment:
I'm guessing we don't want frameless to default to true. Could you change this to false?